src/components/ui/Select/Select.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Select as UISelect } from '@faststore/ui'
+import type { SelectProps } from '@faststore/ui'
+import { CaretDown as CaretDownIcon } from 'phosphor-react'
+
+import './select.scss'
+
+type SelectOptions = {
+  [key: string]: string
+}
+
+interface UISelectProps extends SelectProps {
+  /*
+   * Defines the options available in the select. The SelectOptions object
+   * keys are the property names, while the values correspond  to the text that
+   * will be displayed in the UI
+   */
+  options: SelectOptions
+  /*
+   * Specifies the text that will be displayed in the label right next to the Select.
+   * If omitted, the label will not be rendered.
+   */
+  labelText?: string
+}
+
+export default function Select({
+  options,
+  onChange,
+  labelText,
+  value,
+  'aria-label': ariaLabel,
+  className,
```

Comment:
```suggestion
  classes,
```

Reply:
Just to make clear that are the extra classes that this component might receive. 
Can you also add this on the interface as optional?

`classes?: string`